src/main/scala/org/scalastyle/Main.scala

```diff
@@ -47,6 +48,8 @@ object Main {
     println("     --inputEncoding STRING      encoding for the source files")
     println(" -w, --warnings true|false       fail if there are warnings")
     println(" -e, --externalJar FILE          jar containing custom rules")
+    println(" -x, --excludedFiles STRING      regex matches file paths to exclude")
```

Comment:
I expanded the documentation to describe the use of semicolons to delimit the regexes.   It would probably be useful to include a brief overview and examples on the website or in the README to explain the exlcudedFiles parameter in greater depth.